OrbitLinuxTracing/GpuTracepointEventProcessor.cpp

```diff
@@ -69,7 +69,7 @@ void GpuTracepointEventProcessor::CreateGpuExecutionEventIfComplete(const Key& k
   // at the timestamp of scheduling the current job, we push the start
   // time for starting on the hardware back.
   if (!timeline_to_latest_dma_signal_.contains(timeline)) {
-    timeline_to_latest_dma_signal_.emplace(timeline, dma_it->second.timestamp_ns);
```

Comment:
I would add a short comment why setting to zero here is fine. 

Suggestion:
"When there is not yet an entry in timeline_to_latest_dma_signal_ for the current timeline, this means that no previous GPU job has been executed on this timeline during our capture. We just have to set a timestamp here that lies before any event on the timeline to make sure that the hardware start time below is set correctly, so "0" is a perfectly valid choice."

Reply:
Done and thanks for giving a suggestion (just changed a couple of words).

Reply:
Looks great, thanks!